src/createServer.js

```diff
@@ -1,3 +1,55 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const urlArr = req.url.split('?');
+    const text = urlArr[0].slice(1);
+    const params = new URLSearchParams(urlArr[1]);
+    const toCase = params.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = [];
+
+    if (!text) {
+      errors.push({
+        message: 'Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message: '"toCase" query param is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (
+      !['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(toCase)
```

Comment:
`['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER']` - looks like separate variable
and move into separate file